packages/calypso-build/babel.config.js

```diff
@@ -0,0 +1,57 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				useBuiltIns: 'entry',
+				corejs: 2,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
+			},
+		],
+		'@babel/react',
+	],
+	plugins: [
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
+		'@babel/plugin-syntax-dynamic-import',
+		[
+			'@babel/transform-runtime',
+			{
+				corejs: false, // we polyfill so we don't need core-js
+				helpers: true,
+				regenerator: false,
+				useESModules: false,
+			},
+		],
+	],
+	env: {
+		build_pot: {
+			plugins: [
+				[
+					'@wordpress/babel-plugin-makepot',
+					{
+						output: 'build/i18n-calypso/gutenberg-strings.pot',
+						headers: {
+							'content-type': 'text/plain; charset=UTF-8',
+							'x-generator': 'calypso',
+						},
+					},
+				],
```

Comment:
Technically, we still have a few places in `gutenberg/extensions` (p2 preset) that use translation functions from `@wordpress/i18n`. However, I don't think it makes sense to keep this Babel plugin at global level. Maybe it'll be best to turn the p2 preset into a (pseuo-)package (with its own Babel config).

Reply:
I'm not sure where this should live. Does it make sense as part of the `calypso-build` package? I'm really not sure.

Reply:
I wouldn't really move it there, TBH -- at least not for now. JP blocks is our only precedent for now, and we hope to not use `@wordpress/babel-plugin-makepot` for that (but rely on the plugin repo's string extraction).